procedures/unit-testing-basics.ipf

```diff
@@ -2201,6 +2233,7 @@ static Function CallTestCase(s, reentry)
 		UTF_PrintStatusMessage(msg)
 	else
 		func = s.fullFuncName
+		SaveAssertionCounter()
```

Comment:
Is there  a reason we don't call that in TestCaseBegin? This would be symmetrical compared to the call to GetSavedAssertionCounter.

Reply:
The reason is that it is not symmetrical. By our definition the user hook for test case begin is called after TestCaseBegin in ExecuteHooks.